.github/workflows/ci.yml

```diff
@@ -9,12 +9,13 @@ jobs:
     timeout-minutes: 30
     strategy:
       matrix:
-        python-version: [3.7, 3.9]
-        backend: ['django']
+        python-version: ['3.9', '3.10']
+        backend: ['psql_dos']
         editable_install_option: ['-e', '']
-        exclude:
-        - python-version: 3.7
-          editable_install_option: ''
+        include:
+        - python-version: '3.7'
```

Comment:
This ends up installing `aiida-core` 1.x, right? Might be good to add a comment explaining this.

Reply:
Done